pkgs/development/python-modules/amarna/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, lark
+, pydot
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "amarna";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "crytic";
+    repo = "amarna";
+    rev = "v${version}";
+    sha256 = "sha256-ohR6VJFIvUCMkppqdCV/kJwEmh1fP0QhfQfNu3RoMeU=";
+  };
+
+  propagatedBuildInputs = [
+    lark
+    pydot
+  ];
+  checkInputs = [ pytestCheckHook ];
+  pythonImportsCheck = [ "amarna" ];
+
+  meta = with lib; {
+    description = "Amarna is a static-analyzer and linter for the Cairo programming language.";
```

Comment:
Sounds to me like a cli tool which should live outside of pythonPackages.

Reply:
You can import `amarna` to reuse it elsewhere and build on the top of it. It is not only a linter, but provides API for static analysis.